.project

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>Palladio-Addons-DataFlowConfidentiality-Analysis</name>
```

Comment:
Should this file be in here?

Reply:
Removed in 21740d57b110afe684fbd2855fce39a95394a07c